docs/_component/modal/index.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: modal
+title: modal / dialog
 layout: default
-seo-title: Modal
+seo-title: Modal / Dialog
 description: A third party plugin we use to produce an accessible modal dialog component.
```

Comment:
Since you updated all the titles to include "/ Dialog", make the description consistent as well and update `modal dialog` to `modal / dialog`?